app/models/user.rb

```diff
@@ -2,14 +2,23 @@
 
 # The user class
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Your app works fine but you need to add `:confirmable` in order to be able to send a confirmation email while signing up please make sure to do that 
- Screenshot of the requirements 
<img width="529" alt="Screen Shot 2022-11-09 at 11 08 30" src="https://user-images.githubusercontent.com/61790587/200802507-96e214fd-9c08-4c52-ad30-a788ef7cf03f.png">